app/main.go

```diff
@@ -108,9 +108,6 @@ func Main(config Config) {
 		}
 	}
 
-	if len(config.SHA256Sums) == 0 {
-		config.SHA256Sums = hermit.ScriptSHAs
-	}
```

Comment:
I dropped this check because it duplicates [the behaviour in `OpenEnv`](https://github.com/cashapp/hermit/blob/b745a072420ce26390a35cf8035af4444dd1e115/env.go#L300-L302).